_includes/tutorials/iot-loss-tracking/confluent/code/tutorial-steps/dev/process.sql

```diff
@@ -0,0 +1,28 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create a stream for raw telemetry data
+CREATE STREAM iot_telemetry (
+  device_id INT,
+  ts BIGINT
+) WITH (
+  KAFKA_TOPIC = 'iot_telemetry',
+  PARTITIONS = 1,
+  VALUE_FORMAT = 'AVRO',
+  TIMESTAMP = 'ts'
+);
+
+-- Create lags per device over tumbling window
+CREATE TABLE iot_telemetry_lags WITH (
+  KAFKA_TOPIC = 'iot_telemetry_lags',
+  PARTITIONS = 1,
+  VALUE_FORMAT = 'AVRO'
```

Comment:
@davetroiano this is not required, it inherits from the original topic.  None of the recipes provide this either.

Reply:
removed this